src/test/java/org/pesmypetcare/webservice/service/medalmanager/UserMedalServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package org.pesmypetcare.webservice.service.medalmanager;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.dao.medalmanager.UserMedalDao;
+import org.pesmypetcare.webservice.entity.medalmanager.UserMedalEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.pesmypetcare.webservice.error.DocumentException;
+import org.pesmypetcare.webservice.service.medalmanager.UserMedalService;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redundant import from the same package - org.pesmypetcare.webservice.service.medalmanager.UserMedalService.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517612)